src/containers/shared/components/Streams.jsx

```diff
@@ -139,7 +138,14 @@ class Streams extends Component {
       streams: ['ledger', 'validations'],
     });
 
-    this.mounted = false;
+    if (this.onledgerWrapper) {
```

Comment:
```suggestion
    if (this.onLedgerWrapper) {
```
I think the casing should be fully camel case for this and all other `this.on<Something>` vars.

Reply:
I was using the convention already set in this file what it wraps is called `onledger`.

Reply:
Ah I see, ok 👍